docs/advanced-features/static-html-export.md

```diff
@@ -47,10 +47,6 @@ By default `next export` doesn't require any configuration. It will generate a d
 
 > To learn more about `exportPathMap` please visit the [documentation for the `exportPathMap` API](/docs/api-reference/next.config.js/exportPathMap.md).
 
-## Deployment
-
-You can read about deploying your Next.js application in the [deployment section](/docs/deployment.md).
```

Comment:
Do we really want to remove this section?

Reply:
I thought about it again, let's keep it. Thanks!